debian/control

```diff
@@ -3,7 +3,7 @@ Section: net
 Priority: optional
 Maintainer: Dell EMC <ops-dev@lists.openswitch.net>
 Build-Depends: debhelper (>= 9),dh-autoreconf,dh-systemd,autotools-dev,libopx-common-dev (>= 1.4.0),libopx-nas-common-dev (>= 6.1.0),
-            libopx-cps-dev (>= 3.6.2),libopx-base-model-dev (>= 3.148.0),libopx-logging-dev (>= 2.1.0)
+            libopx-cps-dev (>= 3.6.2),libopx-base-model-dev (>= 3.109.0),libopx-logging-dev (>= 2.1.0)
```

Comment:
Why is the version _lower_? It's going from `>= 3.148.0` to `>= 3.1.09.0`

Reply:
Good catch. These changes should be here. I'll revert this file.